### ISSUE TITLE
No more updates of filter lists in current uBO with RPC default policy "Block requests by default"

### ISSUE THREAD

**Issue by potassiumchloride** [RequestPolicyContinued/requestpolicy#844]
Dear Martin,

I reported this issue originally for uBlock Origin but its developer suggested to contact you instead. There seems to be a problem in the interaction of RPC and uBO as the update mechanism of filter lists in uBO seems to be blocked by RPC.

Please have a look at my original report at:
https://github.com/gorhill/uBlock/issues/2965

**Comment by gorhill** [comment]
Probably a duplicate of #835.

**Comment by potassiumchloride** [comment]
After gorhill's pointer to the open issue https://github.com/RequestPolicyContinued/requestpolicy/issues/835 I also added moz-extension://*:* to the allowed destinations, but unfortunately uBO 1.14.4 doesn't update the filter lists. If you need any further information or more details for the bug report, please let me know.

**Comment by potassiumchloride** [comment]
gorhill from uBO came up with the right idea to solve the issue. To make updates of the filter lists work again in the webext version of uBO one has to activate `moz-extension://*:*` as source instead of destination in the settings of RPC (when the default policy is to block all 3rd party requests). Thus, I assume this should be allowed by default in future versions of RPC. At this point many thanks to gorhill for helping me to solve the issue!

**Comment by myrdd** [comment]
Thank you for reporting this issue. This issue is now moved to the subscription repo: https://github.com/RequestPolicyContinued/subscriptions/issues/61.
